docs/api-docs/tax/tax-rates-zones.md

```diff
@@ -198,6 +198,11 @@ title: Response
 
 <!-- type: tab-end -->
 
+<!-- theme: info -->
+> #### Note
```

Comment:
make the callout title more specific. when they are called "Note" we have a bunch of duplicate anchor links on the page.

Reply:
@slsriehl Callout is now specific to updating a zone